news/2019-07-07-beatmap-spotlights-spring-2019.md

```diff
@@ -0,0 +1,306 @@
+
+As always, the map has a lot of contrast, with every separate section of the map feeling a unique and close match to the features of the song. The hitsounding, also done by AIR, follows the same idea, adding to the experience. If the top difficulty is a bit too jumpy, be sure to check out the Insane and Hard which maintain the same concepts as the top difficulty, but at a more manageable level.
+
+_written by [Nao Tomori](https://osu.ppy.sh/users/5364763)
+
+[![](/wiki/shared/news/2019-07-07-beatmap-spotlights-spring-2019/osu/kyoumen.jpg)](https://osu.ppy.sh/beatmapsets/731108)
+
+Like last time, we present you another set of "Kyoumen no Nami" by YURiKA. This time it's [Vell](https://osu.ppy.sh/users/3414541)'s turn to show what amazing things you can do with this breathtaking song. He captures the song's beauty perfectly with his movement choices and hitsounds that the map is definitly a hightlight to play. [Darky1](https://osu.ppy.sh/users/3372928)'s storybeard definitly adds to the atmosphere for a great gameplay experience.
+
+Don't forget to check out on the lower difficulties by [Stjpa](https://osu.ppy.sh/users/2954693) and [Smokeman](https://osu.ppy.sh/users/2140676) which both also archieve to follow the song in a very enjoyable way.
+
+_written by [Icekalt](https://osu.ppy.sh/users/5410645)
+
+[![](/wiki/shared/news/2019-07-07-beatmap-spotlights-spring-2019/osu/focus.jpg)](https://osu.ppy.sh/beatmapsets/925329)
+
+Capturing the essence of a song can be considered a herculean feat for a mapper. This map gets the job done. [Vell](https://osu.ppy.sh/users/3414541) once again takes off the brakes and delivers another masterpiece of contemporary mapping with sukekiyo's song "focus". In tune with the music along [Darky1](https://osu.ppy.sh/users/3372928)'s storyboard, the map follows the baroquian motif of vanitas through which it blooms into a thundering roar of raw emotion to then just as quickly fades away into nothingness. All the while, it showcases pristine mapping along with the natural melodic texture.
+
+Do not deprive yourself of such an experience.
+
+_written by [Smokeman](https://osu.ppy.sh/users/2140676)
+
+[![](/wiki/shared/news/2019-07-07-beatmap-spotlights-spring-2019/osu/line.jpg)](https://osu.ppy.sh/beatmapsets/899381)
+
+A sharp contrast from recent trends of electronic and anime music, [Yuuya_ke](https://osu.ppy.sh/users/6441720) delivers a strikingly simplistic, yet effective mapset of Ricky Montgomery's "Line Without A Hook". The highly emotive indie track is accented beautifully through a level of flow and rhythm mastery unprecedented from most mappers' first ranked mapsets. Regardless, Line Without A Hook doesn't disappoint in highlighting how even rudimentary patterning can culminate into a moving, well-polished product that'll leave you begging for more.
```

Comment:
```suggestion
A sharp contrast from recent trends of electronic and anime music, [Yuuya_ke](https://osu.ppy.sh/users/6441720) delivers a strikingly simplistic, yet effective mapset of Ricky Montgomery's "Line Without A Hook". 

The highly emotive indie track is accented beautifully through a level of flow and rhythm mastery unprecedented from most mappers' first ranked mapsets. Regardless, Line Without A Hook doesn't disappoint in highlighting how even rudimentary patterning can culminate into a moving, well-polished product that'll leave you begging for more.
```

Reply:
this one is also really good btw, good example to show others when writing stuff